doc/config_options.txt

```diff
@@ -1111,7 +1111,15 @@ Possible values are `allow`, `block`, or `managed`.
 - When set to `block`, this option prevents using all disk devices except the root one.
 - When set to `managed`, this option allows using disk devices only if `pool=` is set.
 - When set to `allow`, there is no restriction on which disk devices can be used.
-```
+
+  ```{important}
+  When allowing all disk devices, make sure to set
+  {config:option}`project-restricted:restricted.devices.disk.paths` to a list of
```

Comment:
Thanks!

@ru-fu It'd be good to clarify what "shifted devices" is here (i.e those with `shift=true` set).
Also do you think we should specify on `restricted.devices.disk.paths` that it prevents `shift=true`?

Reply:
@tomponline Does `restricted.devices.disk.paths` explicitly prevent `shift=true`? My understanding was that it just restricts the device path, which in turn means that devices with `shift=true` wouldn't be allowed because they would have a different source path then. If that is true, I don't think we need to specify it again.